browser/main/NoteList/index.js

```diff
@@ -255,6 +255,14 @@ class NoteList extends React.Component {
         .map((uniqueKey) => data.noteMap.get(uniqueKey))
     }
 
+    if (location.pathname.match(/\/tags/)) {
+      return data.noteMap.map(note => {
+        return note
```

Comment:
It seems that note was not edit. Is it necessary?

Reply:
`data.noteMap` has `map` function, but it has not `filter` function.
We can get array of noteObject by using`data.noteMap.map(note => { return note }`.

Reply:
😢